src/main/java/players/mcts/MCTSEnums.java

```diff
@@ -7,7 +7,7 @@
 public class MCTSEnums {
 
     public enum strategies {
-        RANDOM
+        RANDOM, Dominion_BigMoney, Dominion_PlayActions
```

Comment:
This is a strange thing to see in a general player. Can these be in a separate enum if needed for some experiments (or even a Dominion-specific MCTS player)?

Reply:
It's a fair cop. I shudder at the thought of a Dominion-specific MCTS player, but I'm sure there can be a more general method to inject game-specific rollout strategies. (And this is now actually less immediately necessary for experiments, so i'll remove these.)